.github/workflows/publish-docker-cg-producer.yaml

```diff
@@ -0,0 +1,52 @@
+name: Publish Docker Image of Fasten PyCG Producer
+
+on:
+  push:
+    branches:
+      - main
+env:
+  IMAGE_NAME: cg-producer
+
+jobs:
+  push:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Log into registry
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u ${{ github.actor }} --password-stdin
+
+      - name: Download PyCG
+        working-directory: ./cg-producer
+        run: | 
+          git clone https://github.com/vitsalis/PyCG.git
+
+
+            
+
+      - name: Build Docker image
+        working-directory: ./cg-producer
+        run: docker build -t $IMAGE_NAME .
+
+      - name: Push image
+        run: |
+          IMAGE_ID=docker.pkg.github.com/${{ github.repository }}/$IMAGE_NAME
+          # Change all uppercase to lowercase
+          IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+          
+          # Strip git ref prefix from version
+          VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+          # Strip "v" prefix from tag name
+          [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
+          # Use Docker `latest` tag convention
+          [ "$VERSION" == "master" ] && VERSION=${GITHUB_SHA::8}
```

Comment:
Wasn't `master` renamed to `main`?

Reply:
Fixed on [551ae39](https://github.com/fasten-project/pypi-tools/pull/6/commits/551ae3998a000f848c97704948e916a36e41e556)